packages/skeleton-svelte/src/lib/components/Switch/Switch.svelte

```diff
@@ -53,14 +52,15 @@
 	const [snapshot, send] = useMachine(
 		zagSwitch.machine({
 			id: useId(),
-			name,
-			onCheckedChange(details) {
-				checked = details.checked;
-			}
+			name
 		}),
 		{
 			context: {
 				...zagProps,
+				onCheckedChange: (details) => {
+					checked = details.checked;
```

Comment:
Please swap the order of operations around, first invoke the callback, then sync the binding. This needs to be consistent across all the components. For the rest this looks good!